app/digital/ts/models/ioobjects/gates/ORGate.ts

```diff
@@ -24,6 +27,52 @@ export class ORGate extends Gate {
         return "or.svg";
     }
 
+    public getMinPos(): Vector {
+        const min = V(Infinity);
+
+        // Find minimum pos from corners of transform
+        var BOX_WIDTH = DEFAULT_BORDER_WIDTH;
+        if(this.numInputs() >3 && this.numInputs() < 7)
+            BOX_WIDTH += 50;
+        if(this.numInputs() == 8)
+            BOX_WIDTH += 100;
```

Comment:
Let's try and find a constant or equation to make this more defined rather than having magic numbers.

If there's no constant then let's add an appropriately named one.